Testlio-Automation/testlio-automation-templates
# GREEN-702: Add granular video recording example for Node

#### JIRA
[GREEN-702](https://testlions.atlassian.net/browse/GREEN-702)

#### Related PRs
- https://github.com/Testlio/arc-service/pull/426

#### Background and Solution
* Added example for recording granular videos for Node tests.
* Remove unnecessary Node test spec (updating it to be the default one in S3 for all envs).
* Misc TS typing fixes and removed a large deprecated dependency.

#### Testing Strategy
1. Change the `username` to `testlio` and `password` to `lovetesting` in `testlio-mobile-node-template/config.ts`.
2. Run `npm run package` in the `testlio-mobile-node-template` directory.
3. Grab the `testlio-sample-mobile-app.apk` build and newly generated test package from this repo.
4. Execute an automated run with the build & test package - Google Pixel 2 should be successful.
5. Check in the device drawer that there's a granular video. Under the `Shared` header.

#### Are the Changes Covered With Tests?

#### Does This Introduce a Breaking Change to the API?

#### Is the Documentation Updated/Provided?
